package.json

```diff
@@ -15,8 +15,8 @@
     "build:remote-schemas": "cp src/data/*.graphql build/src/data/",
     "build": "yarn build:lib && yarn build:index && yarn build:fixtures && yarn build:remote-schemas",
     "ci": "yarn test",
-    "deploy-cloudflare-workers:staging": "yarn wrangler deploy --config=workers/test/ciTestWorker.toml --env staging",
-    "deploy-cloudflare-workers": "yarn wrangler deploy --config=workers/test/ciTestWorker.toml --env staging",
+    "deploy-cloudflare-workers:staging": "yarn wrangler deploy --config=workers/caching/mpcache.toml --env staging",
+    "deploy-cloudflare-workers": "yarn wrangler deploy --config=workers/caching/mpcache.toml --env production",
```

Comment:
This is obviously very minimal, but we'll cross this bridge when a second worker is added.